packages/wonder-blocks-dropdown/src/components/single-select.tsx

```diff
@@ -225,99 +208,99 @@ type State = Readonly<{
  * </SingleSelect>
  * ```
  */
-export default class SingleSelect extends React.Component<Props, State> {
-    selectedIndex: number;
-
-    static defaultProps: DefaultProps = {
-        alignment: "left",
-        autoFocus: true,
-        disabled: false,
-        enableTypeAhead: true,
-        error: false,
-        light: false,
-        labels: {
+const SingleSelect = (props: Props) => {
+    const selectedIndex = React.useRef(0);
+    const {
+        children,
+        error = false,
+        id,
+        opener,
+        light = false,
+        placeholder,
+        selectedValue,
+        testId,
+        alignment = "left",
+        autoFocus = true,
+        dropdownStyle,
+        enableTypeAhead = true,
+        isFilterable,
+        labels = {
             clearSearch: defaultLabels.clearSearch,
             filter: defaultLabels.filter,
             noResults: defaultLabels.noResults,
             someResults: defaultLabels.someSelected,
         },
-    };
-
-    constructor(props: Props) {
-        super(props);
-
-        this.selectedIndex = 0;
```

Comment:
We use a ref for this now: 
https://github.com/Khan/wonder-blocks/blob/32b4f84272bf6c7b98e904fb9a87260f6ae467fd/packages/wonder-blocks-dropdown/src/components/single-select.tsx#L212